crates/rspack_plugin_css/tests/fixtures/custom/composes/snapshot/output.snap

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
js side:
```js
"./b.module.css": function (module, exports, __webpack_require__) {
module.exports = {
  "b": "_b-module__b---__c156016816e95195-c15" + " " + "_b-module__b-1---___1d9724eea6aa131c-_1d",
  "b-1": "_b-module__b-1---___1d9724eea6aa131c-_1d",
};
},
"./d.module.css": function (module, exports, __webpack_require__) {
module.exports = {
  "d": "_d-module__d---___34b192ed7623f91e-_34" + " " + "_d-module__d-1---__c54e45bb853e1f4b-c54",
  "d-1": "_d-module__d-1---__c54e45bb853e1f4b-c54",
};
},
"./f.module.css": function (module, exports, __webpack_require__) {
module.exports = {
  "f": "_f-module__f---___141af47168767e48-_14" + " " + "_f-module__f-1---___85bca5b4a297a364-_85",
  "f-1": "_f-module__f-1---___85bca5b4a297a364-_85",
};
},
"./style.module.css": function (module, exports, __webpack_require__) {
module.exports = {
  "chain1": "_style-module__chain1---___3a2c0a3269e6f63c-_3a" + " " + "_style-module__chain2---___980d4cb026fb8411-_98" + " " + "c" + " " + __webpack_require__("./d.module.css")["d"] + " " + "e" + " " + __webpack_require__("./f.module.css")["f"],
  "chain2": "_style-module__chain2---___980d4cb026fb8411-_98" + " " + "e" + " " + __webpack_require__("./f.module.css")["f"],
  "root-class": "_style-module__root-class---__aae4264cfe73c127-aae" + " " + "_style-module__chain1---___3a2c0a3269e6f63c-_3a" + " " + "a" + " " + __webpack_require__("./b.module.css")["b"] + " " + "_style-module__chain2---___980d4cb026fb8411-_98" + " " + "c" + " " + __webpack_require__("./d.module.css")["d"] + " " + "e" + " " + __webpack_require__("./f.module.css")["f"],
};
```

I guess we should add snapshot of js side.

Reply:
Yes, I agree with you. JS part is crucial to make it work.

Reply:
@h-a-n-a I've added snapshot of js part.
Check it please.